lib/mfg.c

```diff
@@ -264,3 +264,18 @@ int switchtec_active_image_index_set(struct switchtec_dev *dev,
 			    sizeof(set), NULL, 0);
 	return ret;
 }
+
+/**
+ * @brief Resume device boot.
+ *        Note that after calling this function, the current
+ *        'dev' pointer is no longer valid. Before making further
+ *        calls to switchtec library functions, be sure to close
+ *        this pointer and get a new one by calling switchtec_open().
```

Comment:
Can we also rework this comment to take into account the device might not exist after calling it?

Reply:
Good idea. Updated.